bin/.preload.sh

```diff
@@ -7,7 +7,7 @@ DFAULTCONFIG="conf/config.properties"
 CUSTOMCONFIG="data/settings/customized_config.properties"
 LOGCONFIG="conf/logs/log-to-file.properties"
 STARTUPFILE="data/startup.tmp"
-DFAULTXmx="-Xmx800m";
+DFAULTXmx="1024m";
```

Comment:
The variable is used if the default and customized configuration file doesn't exist. Also, yeah you're correct that it just have to state the size without the argument.